### ISSUE TITLE
The visit profile shows wrong profile image

### ISSUE THREAD

**Issue by smty2018** [narayan954/dummygram#559]
### Description

the visit profile shows wrong profile image, show the image of the user rather than the person who posted the image,

https://github.com/narayan954/dummygram/assets/74114936/7b51aa9e-bf49-4198-9c9e-8f7b28f9d8a3

### Steps to reproduce

will fix this

**Comment by narayan954** [comment]
> ### Description
> the visit profile shows wrong profile image, show the image of the user rather than the person who posted the image,
> 
>  Recording.2023-06-24.180603.mp4 
> ### Steps to reproduce
> will fix this
> 
> ### Screenshots
> _No response_
> 
> ### Desktop (please complete the following information)
> _No response_
> 
> ### Smartphone (please complete the following information)
> _No response_
> 
> ### Additional information
> _No response_

yes that's a buggy implementation, need a better implementation.
We've hard coded to get the firebase authenticated user profile picture, email or other details, instead if you can fix these, please work on them, thank you.

**Comment by addymistrel** [comment]
Can I work on this issue under GSSOC'23.

**Comment by narayan954** [comment]
> Can I work on this issue under GSSOC'23.

Hi @addymistrel , sorry brother but this issue is already assigned.

**Comment by addymistrel** [comment]
@narayan954 is there any issue yet to be assigned so that I can work for it in your project.

**Comment by narayan954** [comment]
> @narayan954 is there any issue yet to be assigned so that I can work for it in your project.

if you'd check smart , you'd have found easily. luckily this one's open, best of luck! :)

**Comment by addymistrel** [comment]
Ok assign this for me so that I can go with it.

**Comment by narayan954** [comment]
> Ok assign this for me so that I can go with it.

Yes sure, go ahead
